src/Screens/HomeScreen/HomeScreen.tsx

```diff
@@ -28,8 +31,8 @@ const HomeScreen = () => {
   );
 
   useEffect(() => {
-    dispatch(fetchTodos());
-  }, [dispatch]);
+    dispatch(fetchTodos(alert.show("Fetch todos")));
```

Comment:
We need to make dynamic alert depends on our action result. You should create a function (maybe in helpers) which receive MESSAGE prop and then call alert.show(message).